packages/stark-ui/src/modules/minimap/components/_minimap.component.scss

```diff
@@ -0,0 +1,229 @@
+/* ============================================================================== */
+/*  S t a r k   M i n i m a p                                                     */
+/* ============================================================================== */
+/* stark-ui: src/modules/minimap/components/minimap.component.scss */
+
+.stark-minimap {
+  display: flex;
+  flex-flow: row wrap;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: flex-end;
+  padding-top: 8px;
+  margin: 0 15px;
+  & .stark-minimap-dots {
+    & ul {
+      margin: 0 8px 0 0;
+      padding: 0;
+    }
+    & li {
+      display: inline-block;
+      margin: 0;
+      padding: 0 3px 0;
+      line-height: 18px;
+      vertical-align: top;
+    }
+    & i {
+      display: inline-block;
+      width: 4px;
+      height: 4px;
+      background-color: mat-color($primary-dark-text-color);
```

Comment:
Can we move all the `background-color` rules and also those using the `mat-color(...)` to a separate `_minimap-theme.scss` file? So those colors can be customized to make the minimap themable